lib/mocha.js

```diff
@@ -881,7 +881,7 @@ Mocha.prototype.failZero = function (failZero) {
  * @return {Mocha} this
  * @chainable
  */
-Mocha.prototype.passOnFailingTestSuite = function(passOnFailingTestSuite) {
+Mocha.prototype.passOnFailingTestSuite = function (passOnFailingTestSuite) {
```

Comment:
This reminds me, I've been meaning to write a blog post about removing unrelated formatting changes from PRs... quick summary: unrelated changes clutter the Git history and make it harder to review PRs.

```suggestion
Mocha.prototype.passOnFailingTestSuite = function(passOnFailingTestSuite) {
```

At some point we'll start enforcing formatting in this repo anyway.